app/src/components/trace/LatencyText.tsx

```diff
@@ -43,7 +44,12 @@ export function LatencyText({
     >
       {showIcon ? (
         <Text color={color} size={size}>
-          <Icon svg={<Icons.ClockOutline />} />
+          <Icon
+            svg={<Icons.ClockOutline />}
+            css={css`
+              font-size: 1.1em;
```

Comment:
Do you want to align this with the same var as the span status icon?

Reply:
I'm trying to make the second line less prominant.